eng/scripts/Language-Settings.ps1

```diff
@@ -179,3 +179,25 @@ function Update-python-CIConfig($pkgs, $ciRepo, $locationInDocRepo, $monikerId=$
 
   Set-Content -Path $pkgJsonLoc -Value $jsonContent
 }
+
+function GetExistingPackageVersions ($PackageName, $GroupId=$null) {
```

Comment:
nit: I suggest putting the `{` on its own line for large scopes like a function.